### ISSUE TITLE
Pipewire: No sound after connecting to Bluetooth device

### ISSUE THREAD

**Issue by AmeerTaweel** [NixOS/nixpkgs#239890]
### Describe the bug

I can connect to my Bluetooth headset but I get no sound.

### Steps To Reproduce

I'm not sure if you will face the same issue, but here are the steps:

1. Connect Bluetooth headset.
2. Try playing a video.

### Expected behavior

I should get sound.

### Additional context

This issue is not present if I pin `nixpkgs` to `6f95dd4fd050daf017cae2dfeb1cea1ec0e4c1a1`. However, I couldn't find the exact commit where the issue was introduced.

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.35, NixOS, 23.11 (Tapir), 23.11.20230625.07059ee`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.15.1`
 - channels(root): `"nixos-21.11"`
 - nixpkgs: `/nix/store/hmdjvalbmsb9x9wir7xq8y623abjl55w-source`
```

**Comment by AmeerTaweel** [comment]
I found the problematic commit with `git bisect`:

```log
b58599373e67c8b09c16384450f28de965d7afa8 is the first bad commit
```

The commit updates `pipewire` from `0.3.67` to `0.3.68`.

**Comment by AmeerTaweel** [comment]
Can you help with this @Kranzes and @K900? (Sorry to bother you but I see you're the maintainers of `pipewire`)

**Comment by K900** [comment]
There are some fixes for Pipewire Bluetooth stuff on staging-next which should be merged in a few days. If that doesn't help, I'd recommend asking in #pipewire:matrix.org on Matrix - feel free to ping me there.

**Comment by AmeerTaweel** [comment]
Thanks for your help. Can you please share the PR?

**Comment by K900** [comment]
It just got merged to master.

**Comment by AmeerTaweel** [comment]
The Pipewire Bluetooth fixes didn't work for me. Also, people in #pipewire:matrix.org were not able to help as well. I'm waiting for #247108 to be merged. Hopefully it will fix the issue.

**Comment by AmeerTaweel** [comment]
Now my Bluetooth headset works! But I don't know exactly why. I made a lot of changes since my last comment here:

- Format my hard-drive and re-install NixOS. I used to run both NixOS and Windows 11 (Dual-Boot). However, now I only have NixOS.
- Increased the `stateVersion` due to the re-install.

So I'm still unsure what was the problem and what fixed it. But I will close this issue anyway, since it seems like no one else had the same issue or was able to reproduce it.
